core/src/test/scala/tech/beshu/ror/unit/acl/factory/CoreFactoryTests.scala

```diff
@@ -98,6 +98,84 @@ class CoreFactoryTests extends AnyWordSpec with Inside with MockFactory {
         headers.head should be(Header.Name(NonEmptyString.unsafeFrom("CorpoAuth")))
       }
     }
+    "check policy" when {
+      "allow policy set" in {
+        val config = rorConfigFromUnsafe(
+          """
+            |readonlyrest:
+            |
+            |  access_control_rules:
+            |
+            |  - name: test_block1
+            |    type: allow
+            |    auth_key: admin:container
+            |
+            |  - name: test_block2
+            |    type:
+            |      policy: allow
+            |    auth_key: test:test
+            |
+            |""".stripMargin)
+        inside(createCore(config)) {
+          case Right(Core(acl: AccessControlList, _)) =>
+            val firstBlock = acl.blocks.head
+            firstBlock.name should be(Block.Name("test_block1"))
+            firstBlock.policy should be(Block.Policy.Allow)
+            firstBlock.verbosity should be(Block.Verbosity.Info)
+            firstBlock.rules should have size 1
+
+            val secondBlock = acl.blocks.tail.head
+            secondBlock.name should be(Block.Name("test_block2"))
+            secondBlock.policy should be(Block.Policy.Allow)
+            secondBlock.verbosity should be(Block.Verbosity.Info)
+            secondBlock.rules should have size 1
+        }
+      }
+      "forbid policy set" in {
+        val config = rorConfigFromUnsafe(
+          """
+            |readonlyrest:
+            |
+            |  access_control_rules:
+            |
+            |  - name: test_block1
+            |    type: forbid
+            |    auth_key: admin:container
+            |
+            |  - name: test_block2
+            |    type:
+            |      policy: forbid
+            |    auth_key: test:test
+            |
+            |  - name: test_block3
+            |    type:
+            |      policy: forbid
+            |      response_if_request_forbidden: "you are unauthorized to access this resource"
```

Comment:
maybe `response_message`?
(because the policy has already told us that the matched request is forbidden, WDYT?)